src/components/ToastPositioner.tsx

```diff
@@ -14,7 +14,10 @@ export const ToastPositioner: React.FC<ToastPositionerProps> = ({
   delete rest.in;
 
   return (
-    <div className={className!} style={style}>
+    <div
+      className={typeof className === 'function' ? className({}) : className}
```

Comment:
This is not needed. The `className` is already computed by the `ToastContainer`

Reply:
A little type magic and toastpositioner takes a className as optional string again.